developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables.md

```diff
@@ -39,9 +39,9 @@ The following environment variables are used for Jenkins setup.
 |private_repo_branch| The branch name in your private repo where the Ansible hosts, common.yml and secrets.yml files exist.
 |private_repo_url| The URL of your private repository. After the repository is created, the URL can be obtained from the browser URL or on clicking the **Clone or Download** button on GitHub. It display an https URL.|
 |private_repo_credentials| The ID field value from Jenkins. After the username and password of the private repo are added in the Jenkins global credentials field, it displays an auto generated ID or the user can set a unique ID.|
-|public_repo_branch|This is a unique variable. All the jobs in Jenkins are by default configured to checkout the Jenkins file from this variable as **${public_repo_branch}**. For example, when a value like release-2.7.0 is provided to this variable, the Jenkins jobs checkout the Jenkins file from release-2.7.0 branch, from the URL configured in the job. If this value is set to **refs/tags/release-2.7.0**, then Jenkins checks out the file from the tag **release-2.7.0**. You can change this variable in the Jenkins job configurations and specify a specific branch or tag name. This is useful when you want to run some jobs from a different branch or tag instead of the value mentioned in the variable.|
+|public_repo_branch|This is a unique variable. All the jobs in Jenkins are by default configured to checkout the Jenkins file from this variable as **${public_repo_branch}**. For example, when a value like release-2.8.0 is provided to this variable, the Jenkins jobs checkout the Jenkins file from release-2.8.0 branch, from the URL configured in the job. If this value is set to **refs/tags/release-2.8.0**, then Jenkins checks out the file from the tag **release-2.8.0**. You can change this variable in the Jenkins job configurations and specify a specific branch or tag name. This is useful when you want to run some jobs from a different branch or tag instead of the value mentioned in the variable.|
 |deploy-conf| The name of the library that is used in the Jenkinsfile. This file is used in the global pipeline libraries section. When the Jenkinsfile has this library name, it checks out a couple of common libraries from the URL configured in this section. The Jenkinsfile requires these libraries to run. To avoid writing the same code at multiple places, **common code** is placed in a separate branch and all the Jenkinsfile can use this common code by calling it as a function. If this name is changed, ensure the name in Jenkinsfile is also changed to the new library name|.
```

Comment:
|deploy-conf| This is the name of the library used in the Jenkinsfile for the global pipeline libraries section. When the Jenkinsfile has this library name, it checks out a shared library from the URL configured in this section. The Jenkinsfile requires these libraries to run. To avoid writing the same code at multiple places, **common code** is placed in a separate branch and all the Jenkinsfile can use this standard code by calling it as a function. If this name is changed, ensure the name in Jenkinsfile is also adapted to the new library name|